config/initializers/simple_form.rb

```diff
@@ -141,5 +141,5 @@
   # config.cache_discovery = !Rails.env.development?
 
   # Default class for inputs
-  # config.input_class = nil
+  config.input_class = 'form-control'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.